cupyx/scipy/__init__.py

```diff
@@ -33,3 +49,12 @@ def get_array_module(*args):
         if isinstance(arg, (_ndarray, _spmatrix)):
             return _cupyx_scipy
     return _scipy
+
+
+def __dir__():
+    return __all__
+
+
+def __getattr__(name):
+    if name in submodules:
+        return _importlib.import_module(f'scipy.{name}')
```

Comment:
This is returning SciPy module instead of CuPy, and not raising an error for undefined attributes.